README.md

```diff
@@ -1,96 +1,24 @@
-# GitHub Gems: Driving Open-Source Investments With Data
+# Christopher Akey
```

Comment:
I'd suggest keeping a title that relates to the project, like "Github Gems: Commit and Stars Data Pipeline" or anything else that resonates with you.